usaspending_api/awards/v2/data_layer/orm.py

```diff
@@ -370,8 +368,77 @@ def fetch_business_categories_by_legal_entity_id(legal_entity_id):
     return []
 
 
+def fetch_all_cfda_details(award):
+    # we default to fain because we don't have a good link between award -> fabs
```

Comment:
It's easy to obtain all of the transactions under an award by using the `award_id` field in `transaction_normalized`
That should be faster and more accurate than attempting to use fain and uri which are not guaranteed to be unique for each award